Traceback (most recent call last):
  File "/local/293704/main.py", line 186, in <module>
    y_train = jnp.concatenate((x_train, yf))
  File "/home/yanw/.conda/envs/uncertain/lib/python3.9/site-packages/jax/_src/numpy/lax_numpy.py", line 1891, in concatenate
    arrays_out = [lax.concatenate(arrays_out[i:i+k], axis)
  File "/home/yanw/.conda/envs/uncertain/lib/python3.9/site-packages/jax/_src/numpy/lax_numpy.py", line 1891, in <listcomp>
    arrays_out = [lax.concatenate(arrays_out[i:i+k], axis)
  File "/home/yanw/.conda/envs/uncertain/lib/python3.9/site-packages/jax/_src/lax/lax.py", line 622, in concatenate
    return concatenate_p.bind(*operands, dimension=dimension)
  File "/home/yanw/.conda/envs/uncertain/lib/python3.9/site-packages/jax/_src/core.py", line 420, in bind
    return self.bind_with_trace(find_top_trace(args), args, params)
  File "/home/yanw/.conda/envs/uncertain/lib/python3.9/site-packages/jax/_src/core.py", line 423, in bind_with_trace
    out = trace.process_primitive(self, map(trace.full_raise, args), params)
  File "/home/yanw/.conda/envs/uncertain/lib/python3.9/site-packages/jax/_src/core.py", line 913, in process_primitive
    return primitive.impl(*tracers, **params)
  File "/home/yanw/.conda/envs/uncertain/lib/python3.9/site-packages/jax/_src/dispatch.py", line 87, in apply_primitive
    outs = fun(*args)
  File "/home/yanw/.conda/envs/uncertain/lib/python3.9/site-packages/jax/_src/traceback_util.py", line 179, in reraise_with_filtered_traceback
    return fun(*args, **kwargs)
  File "/home/yanw/.conda/envs/uncertain/lib/python3.9/site-packages/jax/_src/pjit.py", line 248, in cache_miss
    outs, out_flat, out_tree, args_flat, jaxpr, attrs_tracked = _python_pjit_helper(
  File "/home/yanw/.conda/envs/uncertain/lib/python3.9/site-packages/jax/_src/pjit.py", line 136, in _python_pjit_helper
    infer_params_fn(*args, **kwargs)
  File "/home/yanw/.conda/envs/uncertain/lib/python3.9/site-packages/jax/_src/api.py", line 325, in infer_params
    return pjit.common_infer_params(pjit_info_args, *args, **kwargs)
  File "/home/yanw/.conda/envs/uncertain/lib/python3.9/site-packages/jax/_src/pjit.py", line 495, in common_infer_params
    jaxpr, consts, out_shardings, out_layouts_flat, attrs_tracked = _pjit_jaxpr(
  File "/home/yanw/.conda/envs/uncertain/lib/python3.9/site-packages/jax/_src/pjit.py", line 1150, in _pjit_jaxpr
    jaxpr, final_consts, out_type, attrs_tracked = _create_pjit_jaxpr(
  File "/home/yanw/.conda/envs/uncertain/lib/python3.9/site-packages/jax/_src/linear_util.py", line 350, in memoized_fun
    ans = call(fun, *args)
  File "/home/yanw/.conda/envs/uncertain/lib/python3.9/site-packages/jax/_src/pjit.py", line 1089, in _create_pjit_jaxpr
    jaxpr, global_out_avals, consts, attrs_tracked = pe.trace_to_jaxpr_dynamic(
  File "/home/yanw/.conda/envs/uncertain/lib/python3.9/site-packages/jax/_src/profiler.py", line 336, in wrapper
    return func(*args, **kwargs)
  File "/home/yanw/.conda/envs/uncertain/lib/python3.9/site-packages/jax/_src/interpreters/partial_eval.py", line 2314, in trace_to_jaxpr_dynamic
    jaxpr, out_avals, consts, attrs_tracked = trace_to_subjaxpr_dynamic(
  File "/home/yanw/.conda/envs/uncertain/lib/python3.9/site-packages/jax/_src/interpreters/partial_eval.py", line 2336, in trace_to_subjaxpr_dynamic
    ans = fun.call_wrapped(*in_tracers_)
  File "/home/yanw/.conda/envs/uncertain/lib/python3.9/site-packages/jax/_src/linear_util.py", line 192, in call_wrapped
    ans = self.f(*args, **dict(self.params, **kwargs))
  File "/home/yanw/.conda/envs/uncertain/lib/python3.9/site-packages/jax/_src/dispatch.py", line 98, in prim_fun
    return prim.bind(*args, **params)
  File "/home/yanw/.conda/envs/uncertain/lib/python3.9/site-packages/jax/_src/core.py", line 420, in bind
    return self.bind_with_trace(find_top_trace(args), args, params)
  File "/home/yanw/.conda/envs/uncertain/lib/python3.9/site-packages/jax/_src/core.py", line 423, in bind_with_trace
    out = trace.process_primitive(self, map(trace.full_raise, args), params)
  File "/home/yanw/.conda/envs/uncertain/lib/python3.9/site-packages/jax/_src/interpreters/partial_eval.py", line 2008, in process_primitive
    return self.default_process_primitive(primitive, tracers, params)
  File "/home/yanw/.conda/envs/uncertain/lib/python3.9/site-packages/jax/_src/interpreters/partial_eval.py", line 2012, in default_process_primitive
    out_avals, effects = primitive.abstract_eval(*avals, **params)
  File "/home/yanw/.conda/envs/uncertain/lib/python3.9/site-packages/jax/_src/core.py", line 456, in abstract_eval_
    return abstract_eval(*args, **kwargs), no_effects
  File "/home/yanw/.conda/envs/uncertain/lib/python3.9/site-packages/jax/_src/lax/utils.py", line 60, in standard_abstract_eval
    return core.ShapedArray(shape_rule(*avals, **kwargs),
  File "/home/yanw/.conda/envs/uncertain/lib/python3.9/site-packages/jax/_src/lax/lax.py", line 3205, in _concatenate_shape_rule
    raise TypeError(msg.format(dimension, ", ".join(map(str, shapes))))
TypeError: Cannot concatenate arrays with shapes that differ in dimensions other than the one being concatenated: concatenating along dimension 0 for shapes (10, 2), (10, 1).
--------------------------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code. Per user-direction, the job has been aborted.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
mpirun detected that one or more processes exited with non-zero status, thus causing
the job to be terminated. The first process to do so was:

  Process name: [[7237,1],0]
  Exit code:    1
--------------------------------------------------------------------------
cp: missing destination file operand after '*.png/home/yanw/uncertain/results/figures/0617'
Try 'cp --help' for more information.
cp: missing destination file operand after '/home/yanw/uncertain'
Try 'cp --help' for more information.
